src/pages/AdminPage.js

```diff
@@ -0,0 +1,28 @@
+import React,{useState,useEffect} from "react";
+import {Link} from "react-router-dom";
+const AdminPage = () => {
+  const [data,setData]=useState(0);
+
+  return (
+    <div className="center">
+      <Link to="/">
+            <button className="back">Back</button>
+        </Link>
+      <div className="text-in-box">
```

Comment:
I have updated the slider on this page to display the value. I also reorganized the text into a <div> so the body text will be inside a box. The box should be able to scroll when text overflows in the y direction.